adafruit_ssd1306.py

```diff
@@ -130,6 +130,12 @@ def text(self, string, xpos, ypos, col=1):
         """Place text on display"""
         self.framebuf.text(string, xpos, ypos, col)
 
+    #pylint: disable-msg=too-many-arguments
```

Comment:
I think you only need this `disable` once if you put it in the method after the comment.

Reply:
whoops I meant to re-enable it after this block, I saw some other hunk of code that did that.  Copy/pasted a bit too much, fixing...

Reply:
resolved via 4bc78f8a7ffac812b523f3d9d7cba2f8215d658b